packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,38 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [33.0.0]
+
+### Added
+
+- **BREAKING:** Add `messenger` as a constructor option for `AccountTrackerController` ([#4225](https://github.com/MetaMask/core/pull/4225))
+  - Add `AccountTrackerControllerMessenger` type
+- `NftController` now allows `AccountsController:getAccount`, `AccountsController:getSelectedAccount` as messenger actions and subscribes to the `AccountsController:selectedEvmAccountChange` messenger event ([#4221](https://github.com/MetaMask/core/pull/4221))
+- `NftDetectionController` now allows `AccountsController:getSelectedAccount` as a messenger action ([#4221](https://github.com/MetaMask/core/pull/4221))
+- Token price API support for mantle network ([#4376](https://github.com/MetaMask/core/pull/4376))
+
+### Changed
+
+- **BREAKING:** Bump dependency and peer dependency `@metamask/accounts-controller` to `^17.0.0` ([#4413](https://github.com/MetaMask/core/pull/4413))
+- Upgrade `TokenRatesController` to `BaseControllerV2` ([#4314](https://github.com/MetaMask/core/pull/4314))
+- Update Nft controllers to use `selectedAccountId` instead of `selectedAddress` ([#4221](https://github.com/MetaMask/core/pull/4221))
+- `TokenRatesController` uses checksum instead of lowercase format for token addresses ([#4377](https://github.com/MetaMask/core/pull/4377))
```

Comment:
This feels more like it belongs in the "Fixed" category to me, since it reverts a change that was mistakenly made in a previous release. Or, if it does belong in the "Changed" category, should it count as a breaking change? What are your thoughts?

Reply:
I do agree this looks more like a "Fixed". I'll update this.

Reply:
Done here: [f273f6e](https://github.com/MetaMask/core/pull/4413/commits/f273f6e1001589b157deaa1407a9983990d47a30)